test/controllers/ReasonableExcuseControllerSpec.scala

```diff
@@ -190,6 +190,25 @@ class ReasonableExcuseControllerSpec extends SpecBase {
         ))
         status(result) shouldBe BAD_REQUEST
       }
+
+      "the validation is performed against an empty value - value is an empty string" in
+        new Setup(AuthTestModels.successfulAuthResult) {
+        when(mockAppealService.getReasonableExcuseListAndParse()(Matchers.any(), Matchers.any()))
+          .thenReturn(Future.successful(
+            Some(seqOfReasonableExcuses)
+          ))
+
+        val result = controller.onSubmit()(fakeRequestWithCorrectKeys.withJsonBody(
+          Json.parse(
+            """
+              |{
+              |   "value": ""
+              |}
+              |""".stripMargin
+          )
+        ))
+        status(result) shouldBe BAD_REQUEST
```

Comment:
Could you assert what's on the page? Like making sure `"There is a problem"` and `"Select the reason for missing the VAT deadline"` is on the page?
I know it's not really the controllers job - but it saves doing it in the view.